infor-design/enterprise-wc
# 877 - Fix Action Panel lifecycle issues

**Explain the details for making this change. What existing problem does the pull request solve?**
While working on the Svelte examples, some additional guards were needed in IdsActionPanel and its child/slotted components to ensure it loads properly without hydration issues

**Related github/jira issue (required)**:
Closes #877
Related to #792

**Steps necessary to review your pull request (required)**:
All tests should pass on this branch.  When testing http://localhost:4300/ids-action-panel/example.html, no component behavior should have changed from `main`

To test against the Sveltekit examples:
- Pull/Build/Link/Run this branch against [this branch](https://github.com/infor-design/enterprise-wc-examples/compare/main...792-more-sveltekit-examples) of the Sveltekit IDS Web Component Examples
- Open http://localhost:3000/ids-action-panel
- Click the "Example" link (this kicks off hydration as opposed to full page load)
- Open the action panel in the example page.  Its contents should load the same as if a full page load occurred.
